.github/workflows/install_ci.yml

```diff
@@ -29,13 +29,25 @@ jobs:
       - name: Install via command (Linux and macOS)
         if: matrix.os != 'windows-latest'
         run: |
-          v -e "$(curl -fsSL https://raw.githubusercontent.com/v-analyzer/v-analyzer/master/install.vsh)"
+          if [ "${{ github.event_name }}" = "pull_request" ]; then
+            url="https://raw.githubusercontent.com/$GITHUB_ACTOR/v-analyzer/$GITHUB_HEAD_REF/install.vsh"
+          else
+            url="https://raw.githubusercontent.com/$GITHUB_REPOSITORY/$GITHUB_REF_NAME/install.vsh"
+          fi
+          echo "$url"
+          v -e "$(curl -fsSL $url)"
 
       - name: Install via command (Windows)
         if: matrix.os == 'windows-latest'
         shell: msys2 {0}
         run: |
-          v -e "$(curl https://raw.githubusercontent.com/v-analyzer/v-analyzer/master/install.vsh)"
+          if "%GITHUB_EVENT_NAME%" == "pull_request" (
+            url="https://raw.githubusercontent.com/$GITHUB_ACTOR/v-analyzer/$GITHUB_HEAD_REF/install.vsh"
```

Comment:
This is not perfect, if it's forked and the fork is renamed. But I think it's can be good enought